### PR TITLE
Feature/welcome page

### LINKED ISSUE #1773: [Parent] Storybook Improvements
We want to fresh up our Storybook https://patterns.axa.ch/ and show that UX is going hand-in-hand with our technical components. All kinds of customers should consume that Storybook page best.

Thomas created a Figma Scribble to show his "vision":
https://www.figma.com/file/NoDaDyyKkCGueJkJpLNKaC/Storybook?node-id=0%3A1

We cut some sub-task out of that idea to start running with these improvements:

- [x] Update Storybook to the latest version https://github.com/axa-ch/patterns-library/issues/1774
- [ ] Design/create Welcome Page https://github.com/axa-ch/patterns-library/issues/1775
- [x] Create `Latest Releases/Whats new` section with dynamic content https://github.com/axa-ch/patterns-library/issues/1776
- [x] Reorder our components to a flat list https://github.com/axa-ch/patterns-library/issues/1777
- [ ] Create `Development Guide` Page https://github.com/axa-ch/patterns-library/issues/1778

### LINKED ISSUE #1775: [Child of #1773] Design/create Welcome Page
Our welcome page should display some useful information. check out the scribble here:https://www.figma.com/file/NoDaDyyKkCGueJkJpLNKaC/Storybook?node-id=0%3A1

### LINKED ISSUE #1778: [Child of #1773] Create `Development Guide` Page
We want a new page that inlines the Figma presentation 
image

There is a figma plugin for storybook - that could be a solution.

### PR DESCRIPTION
**Pull request by 01231**
Fixes #1775, fixes #1778 and fixes #1773

# Done is when (DoD):
- [ ] Modifications within components are reflected by tests.
- [ ] A self-review of the code has been performed.
- [ ] The modified component properties have been added to the typescript typings.
- [ ] Potential design changes have been reviewed by a designer.
- [ ] The modifications are well documented (README.md, etc.) and showcased in the stories.
- [ ] The modifications are documented in the code, particularly in hard-to-understand areas (focus on **WHY**).
- [ ] The modifications are shortly added to CHANGELOG.md with the issue number.
- [ ] The modifications still work in IE.
- [ ] The modifications are according to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)

### CODE REVIEW

File: src/other/pages/landingpage/story.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        >
+          <axa-button-link
+            href="https://github.com/axa-ch/patterns-library"
+            size="large"
+            style="padding-right:25px;"
+            icon='<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="#fff" d="M12 0c-6.626 0-12 5.373-12 12 0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23.957-.266 1.983-.399 3.003-.404 1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.23.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576 4.765-1.589 8.199-6.086 8.199-11.386 0-6.627-5.373-12-12-12z"/></svg>'
+            >GITHUB</axa-button-link
+          >
+          <axa-button-link
+            href="/?path=/story/guides-getting-started--getting-started"
```
Comment (danyball) [inline src/other/pages/landingpage/story.js]: Link is not working

File: src/other/pages/landingpage/story.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          </div>
+        </axa-commercial-hero-banner>
+        <img
+          class="landingpage__commercial-hero-banner-image"
+          src="axa-design-system-illustration.png"
+          alt="Design system illustration"
+        /><img />
+      </div>
+    </div>
+    ${contact}
```
Comment (danyball) [inline src/other/pages/landingpage/story.js:76]: "Get in touch" Button shows the story in fullscreen. Without panel on the left.

File: src/other/pages/what-is-new/story.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  return formattedMessage;
+};
+
+const getDateFromGitCommit = answerJson => {
+  const formattedDate = new Date(answerJson.items[0].commit.author.date);
+  return formattedDate.toLocaleString(navigator.language);
+};
+
+export default {
+  title: 'What is new',
```
Comment (danyball) [inline src/other/pages/what-is-new/story.js]: The structure is not really cool: What is new > What is new. Maybe move "Whats new" to "Welcome" directory?

**Review by danyball — COMMENTED** [review]
Good work!!

Are the images at this page are covered by licence? These images are critical, because you can see real people there. --> https://axa-ch.github.io/plib-feature/feature/welcome-page/?path=/story/others-contact--contact

File: src/other/pages/guides/getting-started/index.scss [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+.callout__first-element {
+    margin-top: 2rem;
+}
+
+pre {
+    margin: 3rem 0;
+}
+
+// Duplicate to src/other/pages/utils/contact-footer/index.scss
```
Comment (raphaellueckl) [inline src/other/pages/guides/getting-started/index.scss:61]: why duplicate?

File: src/other/pages/guides/getting-started/story.jsx [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+import { AXAText, AXAHeading } from './plib-components';
+import styles from './index.scss';
+
+export default {
+  title: 'Guides/Getting started',
+  decorators: [],
+  parameters: {
+    knobs: { disabled: true },
+    changelog: { disabled: true },
+    codepreview: { disabled: true },
```
Comment (raphaellueckl) [inline src/other/pages/guides/getting-started/story.jsx:14]: Changelog and codepreview do no longer exist

Comment (danyball) [reply src/other/pages/guides/getting-started/story.jsx:14]: No, they are just deactivated. Hope to get them running soon.

Comment (01231) [reply src/other/pages/guides/getting-started/story.jsx:14]: This is on purpose, they are not needed and the pages look better without. I'm trying to not show the addons bar completely.

File: src/other/pages/guides/getting-started/index.scss (hunk 4 above)
Comment (01231) [reply src/other/pages/guides/getting-started/index.scss:61]: Already told the team in the daily. It was not possible for me to use the contact and callout Ad-hoc component in the jsx story.

File: src/other/pages/what-is-new/story.js (hunk 3 above)
Comment (01231) [reply src/other/pages/what-is-new/story.js]: done

File: src/other/pages/landingpage/story.js (hunk 2 above)
Comment (01231) [reply src/other/pages/landingpage/story.js:76]: done

File: src/other/pages/landingpage/story.js (hunk 1 above)
Comment (01231) [reply src/other/pages/landingpage/story.js]: seems not to work in the feature-branch deployment, locally it is working

File: src/other/pages/guides/getting-started/story.jsx (hunk 5 above)
Comment (01231) [reply src/other/pages/guides/getting-started/story.jsx:14]: hide panels done

File: src/other/pages/landingpage/story.js (hunk 1 above)
Comment (markus-walther) [reply src/other/pages/landingpage/story.js]: This should be `window.location.href.replace(/\/[^\/]*$/, '') + '/?path=/story/guides-getting-started--getting-started'`, to adapt to any kind of deployment.

Comment (01231) [reply src/other/pages/landingpage/story.js]: Thanks a lot @markus-walther

File: src/other/pages/guides/getting-started/story.jsx (hunk 5 above)
Comment (danyball) [reply src/other/pages/guides/getting-started/story.jsx:14]: It seems to not work correctly? On initial start of SB theres still "nothing found". But if i open up another story and go back to "welcome" story, the panel is gone?!

File: src/other/pages/landingpage/story.js (hunk 1 above)
Comment (01231) [reply src/other/pages/landingpage/story.js]: done

**Comment by 01231** [comment]
@danyball I changed to no copyright images, I was not sure how the old images were protected.

File: src/components/00-materials/story.js [hunk 6]
```diff
@@ -36,12 +36,13 @@ const mapToIconItem = icon => {
 };
 
 export default {
-  title: 'Others/Materials',
+  title: 'Brand Elements/Icons and Images',
```
Comment (raphael-suter) [inline src/components/00-materials/story.js:39]: How about "brand identity" ? :)

Comment (01231) [reply src/components/00-materials/story.js:39]: With `brand identity` also values etc. of a brand  would be included. `Brand Elements` just includes the core elements which would be a bit more limiting. But I don't have a strong opinion about this :)

**Review by raphaellueckl — APPROVED** [review]
lgtm

**Pull request merged**